certification/internal/shell/has_prohibited_packages.go

```diff
@@ -79,23 +106,23 @@ func (p *HasNoProhibitedPackagesCheck) Help() certification.HelpText {
 // prohibitedPackageList is a list of packages commonly present in the RHEL contianer images that are not redistributable
 // without proper licensing (i.e. packages that are not under the same availability as those found in UBI).
 // TODO: Confirm these packages are the only packages in immediate scope.
```

Comment:
This likely needs a reword now that this is a map.

Reply:
Reworked it slightly.